server.go

```diff
@@ -54,6 +68,7 @@ func (e *ErrResponse) Render(w http.ResponseWriter, r *http.Request) error {
 }
 
 func (link *Link) Render(w http.ResponseWriter, r *http.Request) error {
+	w.Header().Set("Location", link.URL)
```

Comment:
Does this set the `Location` header on every response? Do we want to do that? Also does it blow up when we pass `&Link{}` to a render?

Reply:
Yeah you're right, we should move it to just the `get` I suppose.